src/adapter/scriptSkipper.ts

```diff
@@ -87,10 +91,15 @@ export class ScriptSkipper {
     return this._isUrlSkippedMap.get(url)!;
   }
 
-  public setAllNodeInternalsToSkip(nodeInternalsNames: string[]): void {
-    let fullLibNames = nodeInternalsNames.map(name => name + '.js');
-    fullLibNames.push('^internal/.+\.js|');
-    this._nodeInternalsRegex = this._createRegexString(fullLibNames);
+  public setAllNodeInternals(nodeInternalsNames: string[]): void {
+    this._allNodeInternals = nodeInternalsNames.map(name => name + '.js');
+  }
+
+  public toggleSkipFileStatus(url: string | undefined): void {
```

Comment:
Since it's not meaningful to pass in undefined to this method, I'd prefer to drop the undefined. Mentally it's like having an extra function overload that takes `undefined` and does nothing with it.

Reply:
Yep that makes sense. Fixed